orchestrate.py

```diff
@@ -156,6 +169,47 @@ def post(self):
     def pool(self):
         return self.settings['pool']
 
+class APIPoolHandler(BaseHandler):
+    #@web.authenticated
+    @gen.coroutine
+    def post(self):
+        '''Allow updating pool configuration remotely'''
+        # TODO: Auth
+        # TODO: Determine payload format
+        model = self.get_json_body()
+        if model is None:
+            return self.write({'error': 'empty payload'})
+
+        if 'pool' in model and 'capacity' in model['pool']:
+            new_capacity = int(model['pool']['capacity'])
+            self.pool.capacity = new_capacity
+            yield self.pool.heartbeat()
```

Comment:
There's probably a better (stricter) way to validate the JSON structure. I'd be surprised if there wasn't a library for it somewhere.

Also, I'd argue it should return a 400 for missing required properties. It's a bit weird to me that the `POST` degrades to "give me the current status" with no error if you post valid JSON to it.